spring-cloud-starter-stream-processor-header-enricher/README.adoc

```diff
@@ -0,0 +1,25 @@
+//tag::ref-doc[]
+= Header Enricher Processor
+Use the header-enricher app in a stream to add message headers.
```

Comment:
As I said in other place: no `stream` word I guess for this kind of applications. 
No? Am I missing something?
Thanks